fhd_utils/l_m_n.pro

```diff
@@ -0,0 +1,26 @@
+pro l_m_n, obs, psf, obsdec=obsdec, obsra=obsra, dec_arr=dec_arr, ra_arr=ra_arr,$
```

Comment:
I suggest making this a `function` instead

Reply:
done